src/clientenvironment.cpp

```diff
@@ -206,6 +207,10 @@ void ClientEnvironment::step(float dtime)
 	while(dtime_downcount > 0.001);
 
 	//std::cout<<"Looped "<<loopcount<<" times."<<std::endl;
+	int player_immortal = 0;
+	if(lplayer->getCAO()){
```

Comment:
space

Reply:
also since you are using the CAO again why not put it in a variable? Calling the getter twice feels a bit ugly to me.

Reply:
Nevertheless, code style needs tweaking. i.e. brace removal and space after `if`